### PR TITLE
Add Menu page object

### PR DESCRIPTION
**Pull request by jdpapiero**
- Add Menu page object
- Initialize in LoadPage
- Call initialize in StepsDefinitionSSID

### CODE REVIEW

**Review by marcelogaray — CHANGES_REQUESTED** [review]
please review my observation

**Comment by jdpapiero** [comment]
The observations are reviewed and updated

File: src/main/java/org/umssdiplo/automationv01/core/managepage/Menu/Menu.java [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+    public Menu(){
+        CommonEvents.isVisible(menuStructure);
+    }
+
+    public SubMenuOrganizationalStructure clickMenuOrganizationalStructure() {
+        CommonEvents.clickButton(menuStructure);
+        return new SubMenuOrganizationalStructure();
+    }
+
```
Comment (marcelogaray) [inline src/main/java/org/umssdiplo/automationv01/core/managepage/Menu/Menu.java]: remove blank lines

File: src/main/java/org/umssdiplo/automationv01/core/managepage/Position/Position.java [hunk 2]
```diff
@@ -0,0 +1,15 @@
+package org.umssdiplo.automationv01.core.managepage.Position;
+
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.umssdiplo.automationv01.core.managepage.BasePage;
+import org.umssdiplo.automationv01.core.utils.CommonEvents;
+
+public class Position extends BasePage {
+    @FindBy(xpath = "//mat-paginator")
```
Comment (marcelogaray) [inline src/main/java/org/umssdiplo/automationv01/core/managepage/Position/Position.java]: improve your locator

File: src/main/java/org/umssdiplo/automationv01/core/managepage/Position/Position.java [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+    private WebElement paginator;
+
+    public Position(){
```
Comment (marcelogaray) [inline src/main/java/org/umssdiplo/automationv01/core/managepage/Position/Position.java]: is it a page object? if yes, then you should extend BasePage

File: src/main/java/org/umssdiplo/automationv01/core/utils/LoadPage.java [hunk 4]
```diff
@@ -2,11 +2,16 @@
...
+import org.umssdiplo.automationv01.core.managepage.Menu.Menu;
 
 public final class LoadPage {
     public static Login loginPage() {
         ManageDriver.getInstance().getWebDriver()
                 .navigate().to(PropertyAccessor.getInstance().getBaseUrl());
         return new Login();
     }
+
+    public static Menu menuPage() {
```
Comment (marcelogaray) [inline src/main/java/org/umssdiplo/automationv01/core/utils/LoadPage.java]: remove it

**Review by marcelogaray — CHANGES_REQUESTED** [review]
review observations

File: src/main/java/org/umssdiplo/automationv01/core/managepage/Menu/Menu.java (hunk 1 above)
Comment (jdpapiero) [reply src/main/java/org/umssdiplo/automationv01/core/managepage/Menu/Menu.java]: blank lines are removed

File: src/main/java/org/umssdiplo/automationv01/core/managepage/Position/Position.java (hunk 2 above)
Comment (jdpapiero) [reply src/main/java/org/umssdiplo/automationv01/core/managepage/Position/Position.java]: change locator to title of page

File: src/main/java/org/umssdiplo/automationv01/core/managepage/Position/Position.java (hunk 3 above)
Comment (jdpapiero) [reply src/main/java/org/umssdiplo/automationv01/core/managepage/Position/Position.java]: the class extends of BasePage, the line observed is a constructor of page object

File: src/main/java/org/umssdiplo/automationv01/core/utils/LoadPage.java (hunk 4 above)
Comment (jdpapiero) [reply src/main/java/org/umssdiplo/automationv01/core/utils/LoadPage.java]: the lines are removed

**Review by marcelogaray — APPROVED** [review]

**Pull request merged**